pkg/command/command.go

```diff
@@ -264,9 +271,14 @@ func (c *Command) runCmd(cmd *exec.Cmd) error {
 	}
 
 	if err != nil {
+		if exitErr, ok := err.(*exec.ExitError); ok {
+			if status, ok := exitErr.Sys().(syscall.WaitStatus); ok {
```

Comment:
Does that work in a platform independent way?

In case the "downcast" does not work we will end-up with exit code `0` (the default value for `int`). That might be confusing (Having an error (`cmd.Run() failed`) returned together with an exit code of `0`). Maybe we should consider returning `1` which is some kind of catch-all exit code or - even better IMO - an exit code `-1` or above `255` since this indicates clearly that the exit code cannot come from the shell since those values are out of range. Anyway we should not end up with exit code `0` in case the "downcast"  fails.

Reply:
From what I understood is that this should work on any OS.

Nonetheless, good point, I guess we can just default to `1` then ...

Will add this ...